src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,123 @@
+package ui;
+
+import task.TaskList;
+import task.Task;
+import storage.Storage;
+import parser.Parser;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Prompts users for input and also to output responses to them.
+ */
+public class Ui {
+
+    private static Scanner input = new Scanner(System.in);
+
+    /**
+     * Greets the user when the chatbot just starts and ask them for an input.
+     */
+    public static void greet() {
+
+        System.out.println("Hello! I'm Tako! What can I do for you?\n");
+
+        promptInput();
+    }
+
+    /**
+     * Says goodbye to the user
+     */
+    public static void exit() {
+        System.out.println("Bye! Have a nice day!");
+    }
+
+    /**
+     * Prints out the list of tasks that the chatbot is keeping track of and
+     * ask them for input after that.
+     */
+    public static void printList() {
+        System.out.println("Here are your list of tasks:");
+        for (int i = 0; i < TaskList.length(); i++) {
+            System.out.println(TaskList.listTask(i));
+        }
+        promptInput();
+    }
+
+    /**
+     * Prints out the message when the user added a task to the list and
+     * ask them for input after that.
+     *
+     * @param task task that is being added to the list.
+     */
+    public static void addTaskMessage(Task task) {
```

Comment:
Perhaps a name like `printAddTaskMessage` would make it clearer that the method's purpose is to print something, rather than being confused with 'adding a task message'?